torch/_dynamo/repro/after_aot.py

```diff
@@ -225,6 +227,38 @@ def inner_debug_fn(real_inputs):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 
 
+def maybe_fbcode_instructions():
+    if is_fbcode:
+        extra_deps_formatted = "\n".join([f'        "{dep}",' for dep in extra_deps])
+        if len(extra_deps_formatted) > 0:
+            extra_deps_formatted = "\n" + extra_deps_formatted
+        return f"""\
+\"\"\"
```

Comment:
nit: just use triple single quotes.

```suggestion
        return f'''\
"""
```